packages/web-app-files/src/components/Search/List.vue

```diff
@@ -235,6 +237,16 @@ export default defineComponent({
       return term
     }
 
+    const breadcrumbs = computed(() => {
+      return [
+        {
+          text: !unref(searchTerm)
```

Comment:
Why the `!`? 😅  You could easily omit it and flip the order of the two options... which would make it easier to read because there wouldn't be a negation.

Reply:
Yep you're right 😄